musicHelper.md

```diff
@@ -0,0 +1,128 @@
+#Piano Music Helper
```

Comment:
It is always better to add an extra space after markdown layout symbols

Reply:
Ok, I did it